tests/eru.Application.Tests/Substitutions/Commands/UploadSubstitutionsCommandTests.cs

```diff
@@ -21,7 +22,7 @@ public async Task IsToStringOnRequestWorkingCorrectly()
         {
             var request = new UploadSubstitutionsCommand
             {
-                IpAddress = "8.8.8.8",
+                IpAddress = "198.51.100.1",
```

Comment:
You should store this IP in some static class so its obvious it it for testing. Also this test should not be async.